winfetch.ps1

```diff
@@ -64,7 +64,7 @@ param(
     [switch][alias('g')]$genconf,
     [string][alias('c')]$configpath,
     [switch][alias('n')]$noimage,
-    [switch][alias('l')]$switchlogo,
+    [string][alias('o')]$logo,
```

Comment:
@jcwillox, the reason behind this is because the old switchlogo parameter used an `l` and I assume we want to differentiate between it (just in case anyone is calling Winfetch via a script).

Reply:
This is going to be a breaking change anyway, and it'll break scripts regardless as PowerShell will either say the flag is missing or assume that `-l` is short for `-logo` and still error out unless you provide a valid OS. At least on PSv7 `-l` and `-o` are treated as the same, so the added `-o` is unnecessary.

Reply:
I agree. The `o` is unnecessary. In fact all the single letter aliases are redundant (as of now), because PS allows you to call them as long as they are unambiguous.

Reply:
I've adjusted it back to an `l`, @jcwillox and @rashil2000. My apologies for the delay.